scripts/protocgen-pulsar.sh

```diff
@@ -12,8 +12,10 @@ protoc_install_gopulsar
 echo "Cleaning API directory"
```

Comment:
We might need such script for swagger now too

Reply:
Done https://github.com/cosmos/cosmos-sdk/pull/15442/commits/c97c226b679cc4795f370a3d58b5616f69bb8e19